plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -50,6 +53,11 @@
       - GitLab server enforces minimum password length to 8, set this value with 8 or more characters.
       - Required only if C(state) is set to C(present).
     type: str
+  reset_password:
+    description:
+      - Wether the user can change its password or not.
+    default: False
+    type: bool
```

Comment:
This is needed for all new options:
```suggestion
    type: bool
    version_added: 3.2.0
```